Router/Router/View/LoginViewController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  LoginViewController.swift
+//  Router
+//
+//  Created by Vladka on 28.12.2022.
+//
+
+import UIKit
+
+/// Авторизация
+final class LoginViewController: UIViewController {
+    // MARK: - Private Property
+    private var router: LoginRouter?
+    
+
+    // MARK: - Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        router = LoginRouter(viewController: self)
+    }
+    
+    // MARK: - IBAction
+    
+    @IBAction func signIn(_ sender: Any) {
+        router?.toMainVC()
+    }
+    
+    @IBAction func resetPassword(_ sender: Any) {
```

Comment:
```suggestion
    @IBAction func resetPassword(_ sender: Any) {
```
Я бы добавила слово Action во все кнопки